model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,55 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash
+    attributes:
+      - ref: os.name
+        requirement_level: "required"
+        brief: >
+            This attribute indicates the operating system name of the device on which the crash event occurred.
+        examples: ['iOS', 'Android']
+      - ref: ios.state
+        requirement_level: recommended
+        brief: >
+          This attribute represents the state the application has transitioned into at the occurrence of the event
+          For ios crashes
+        examples: ['active', 'inactive', 'background', 'foreground', 'terminate']
+      - ref: android.state
+        requirement_level: recommended
+        brief: >
+          This attribute represents the state the application has transitioned into at the occurrence of the event
+          For android crashes
+        examples: ['created', 'foreground', 'background']
+      - ref: exception.stacktrace
+        requirement_level: recommended
+        brief: >
+          The contents of printStackTrace() being called on the throwable
+          For android crashes
+      - ref: thread.id
+        requirement_level: recommended
+        brief: >
+          The id of the thread as output by Thread.currentThread().getId()
+          For android crashes
+      - ref: thread.name
+        requirement_level: recommended
+        brief: >
+          The name of the thread as output by Thread.currentThread().getName()
+          For android crashes
+      - id: exception.message
+        requirement_level: recommended
+        brief: >
+          The message within the exception object
+          For android crashes
+        type: string
+        examples: ['java.lang.ArithmeticException: / by zero']
+      - id: ios.crash_report
+        requirement_level: recommended
+        brief: >
+          The contents from PLCrashReporter
+          Contains three primary elements. "images", "threads" and "signal". These could be broken out as individual elements
+          For ios crashes
+        type: string
+        examples: ['*Contents of PL CrashReporter crash report*']
```

Comment:
Can you add a real (maybe shortened) example?

Reply:
So sorry for the very long delay.

Here is a sample of the data generated, with the threads and images arrays collapsed for space.

  "crashReport" : {
    "cpuCount" : 10,
    "appVersion" : "0.9(5)",
    "parentProcessName" : "launchd_sim",
    "faultAddress" : "6751435752",
    "isNative" : "1",
    "processPath" : "\/Users\/mickelsn\/Library\/Developer\/CoreSimulator\/Devices\/DB54D056-8CE5-4841-87A0-44664A7E19DB\/data\/Containers\/Bundle\/Application\/052B1F60-CD75-4978-8C19-7955B0354D0B\/AppDCloudEveryFeature.app\/AppDCloudEveryFeature",
    "cpuLogicalCount" : 10,
    "parentProcessId" : 54459,
    "osBuild" : "23C71",
    "timestamp" : "2024-01-22 20:58:07 +0000",
    "threads" : [

    ],
    "osVersion" : "17.0.1",
    "signalCode" : "TRAP_BRKPT",
    "appMinorVersionAtCrash" : "5",
    "actualTimestamp" : "2024-01-22 20:58:07 +0000",
    "cpuType" : {
      "subType" : "0",
      "cpuType" : "16777228"
    },
    "osVersionAtCrash" : "17.0.1",
    "hardwareModel" : "arm64",
    "processName" : "AppDCloudEveryFe",
    "appBundleId" : "com.AppDynamics.AppDCloudEveryFeature",
    "images" : [
      
	],
    "signalName" : "SIGTRAP",
    "operatingSystem" : 2,
    "appMajorVersionAtCrash" : "0.9",
    "processId" : 99932
  }